crates/napi/src/bindgen_runtime/js_values/bigint.rs

```diff
@@ -107,8 +107,7 @@ impl BigInt {
     if len == 1 {
       (self.words[0] as i128, false)
     } else {
-      let i128_words: [i64; 2] = [self.words[0] as _, self.words[1] as _];
-      let mut val = unsafe { ptr::read(i128_words.as_ptr() as *const i128) };
+      let mut val = self.words[0] as i128 + ((self.words[1] as i128) << 64);
```

Comment:
This change does not change the asm output for little endian arches.
[before](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=721aec636dfd01ac3239c7f23368faeb), [after](https://play.rust-lang.org/?version=stable&mode=release&edition=2021&gist=b1c12a93ebcb034f2796da3b16d86712)